### PR TITLE
Remove reductant code in ChatActivity.kt

### LINKED ISSUE #1218: Few reductant code in ChatActivity.kt
**Actual Behaviour**
Some reductant code in ``` ChatActivity.kt ```

**Expected Behaviour**
Reductant code must be removed.

**Screenshots of the issue**
a
b

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by DipanshKhandelwal**
Fixes #1218

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@arundhati24 @batbrain7 Please review

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -387,11 +386,11 @@ class ChatActivity: AppCompatActivity(), IChatView {
         Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
     }
 
-    fun scrollToEnd(view: View) {
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: No need to remove this , it is required

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 2]
```diff
@@ -387,11 +386,11 @@ class ChatActivity: AppCompatActivity(), IChatView {
...
+    fun scrollToEnd() {
         rv_chat_feed.smoothScrollToPosition(rv_chat_feed.adapter.itemCount - 1)
     }
 
-    fun openSettings(view: View) {
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Same as previous

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 3]
```diff
@@ -408,10 +407,6 @@ class ChatActivity: AppCompatActivity(), IChatView {
         progressDialog.dismiss()
     }
 
-    override fun onBackPressed() {
-        super.onBackPressed();
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: It may be used in future if not called now

**Pull request merged**